packages/leaders-program/src/components/list/nav/contents.vue

```diff
@@ -29,13 +29,17 @@ export default {
       type: Object,
       default: null,
     },
+    hasVideos: {
+      type: Boolean,
+      default: false,
+    },
   },
 
   computed: {
     showVideoIcon() {
       const { youtube } = this;
       if (!youtube) return false;
-      return Boolean(youtube.username || youtube.channelId);
+      return Boolean(youtube.username || youtube.channelId || this.hasVideos);
```

Comment:
@B77Mills and @Shinsina are the `username` and `channelId` checks even needed now? The `item.videos` property is being resolved using the content's `youtubeVideos` field, which would likely be empty if no `username` or `channelId` are present?

Reply:
@zarathustra323   That is correct.  Then we can delete the computed showVideoIcon() and just update the the v-if on line 6 to `v-if="showVideos"`
```

Reply:
@zarathustra323 Appears that they wouldn't. I went to check and make sure this didn't alter previously working entries on PackWorld and Automation World as well. I wasn't sure if there was going to be one where the channelId or username would be set and there wouldn't be any videos.